contributed_definitions/nyaml/NXmpes_xps_liquid.yaml

```diff
@@ -0,0 +1,314 @@
+          scheme(NX_CHAR):
+            enumeration:
+              [
+                "tof",
+                "hemispherical",
+                "double hemispherical",
+                "cylindrical mirror",
+                "display mirror",
+                "retarding grid",
+              ]
+          pass_energy(NX_FLOAT):
+            unit: NX_ENERGY
+            doc: "Energy of the electrons on the mean path of the analyser. Pass energy for hemispherics, drift energy for tofs."
+          energy_scan_mode(NX_CHAR):
+            doc: "Way of scanning the energy axis (fixed or sweep)."
+            enumeration: ["fixed", "sweep"]
+          ENTRANCE_SLIT(NXaperture):
+            exists: recommended
+            doc: "Aperture generating the momentum and/or energy filtering."
+            description(NX_CHAR):
+              doc: "Type of aperture inserted in the beam."
+              enumeration: ["slit", "pinhole", "iris"]
+            shape(NX_CHAR):
+              doc: "Description of the shape of the active part of the aperture, curved or straight for horizontal slits, square or round for pinhole etc."
+              enumeration:
+                [
+                  "curved",
+                  "straight",
+                  "circle",
+                  "square",
+                  "hexagon",
+                  "octagon",
+                  "bladed",
+                ]
+              pass_energy(NX_FLOAT):
+                unit: NX_LENGTH
+                doc: "The relevant dimension for the aperture (slit width, pinhole diameter etc)."
+        ELECTRON_DETECTOR(NXdetector):
+          amplifier_type(NX_CHAR):
+            exists: recommended
+            doc: "Type of electron amplifier in the first amplification step."
+            enumeration: ["MCP", "channeltron"]
+          detector_type(NX_CHAR):
+            doc: "Description of the detector type."
+            enumeration:
+              [
+                "DLD",
+                "Phosphor+CCD",
+                "Phosphor+CMOS",
+                "ECMOS",
+                "Anode",
+                "Multi-anode",
+              ]
+          DATA(NXdata): # Raw signal without calibrated axes.
+            exists: recommended
+            \@signal(NX_CHAR):
+              enumeration: ["raw"]
+            raw(NX_NUMBER): # There is a block of numbers named raw.
+              doc: "Raw data before calibration."
+        GEOMETRY(NXtransformations):
+          exists: optional
+          doc: "Transformations describing the transformation chain of all relevant vectors in the experiment."
+          gravity_axis(NX_NUMBER):
+            doc: "Direction of gravity."
+            \@vector(NX_NUMBER):
+              doc: "The vector in the direction of gravity."
+            \@depends_on(NX_CHAR):
+              doc: "Set to '.' to specify the normal vector to which all other vectors are related."
+          beam_axis(NX_NUMBER):
```

Comment:
Typically, in NeXus the McStas coordinate system is chosen. There, the beam_axis is always along the z-axis. What is the intention here to state it in the file?